### PR TITLE
job file to use login shell

### LINKED ISSUE #2424: cylc job script to use "bash -l" instead of sourcing profiles
It is only legacy reason that we have our job scripts sourcing profiles instead of calling the login shell (`#!/usr/bin/bash -l` or `#!/usr/bin/ksh -l`).

From discussion with @hjoliver via email:
> Our current use of `/etc/profile` and `.profile` is technically incorrect ...  Fixing it could result in a change in behavior for users with a `.bash_profile` who configured stuff for cylc in `.profile` just because they know our job scripts source it.  But we could warn about that in next release.

### PR DESCRIPTION
**Pull request by matthewrmshin**
Close #2424.

### CODE REVIEW

**Comment by matthewrmshin** [comment]
Doc updated.

File: doc/src/cylc-user-guide/cug.tex [hunk 1]
```diff
@@ -526,20 +526,23 @@ \subsubsection{Configure Site Environment on Job Hosts}
 If your users submit task jobs to hosts other than the hosts they use to run
 their suites, you should ensure that the job hosts have the correct environment
 for running cylc. A cylc suite generates task job scripts that normally invoke
-\lstinline=bash=. The job will attempt to source the first of these files it
-finds to set up its environment:
+\lstinline=bash -l=, i.e. it will invoke bash as a login shell to run the job
+script. Users and sites should ensure that their bash login profiles are able
+to set up the correct environment for running cylc and their task jobs.
+
+Your site administrator may custom the environment for all task jobs by adding
```
Comment (oliver-sanders) [inline doc/src/cylc-user-guide/cug.tex]: *customise

Comment (matthewrmshin) [reply doc/src/cylc-user-guide/cug.tex]: Fixed typo.

**Comment by matthewrmshin** [comment]
(Merge as all green with double review approval.)

**Pull request merged**